dev-libs/wlc/wlc-0.0.2-r1.ebuild

```diff
@@ -28,6 +30,7 @@ RDEPEND="virtual/opengl
 			 x11-libs/xcb-util-image
 			 x11-libs/xcb-util-wm
 			 x11-libs/libXfixes )
+		xwayland? ( x11-base/xorg-server[wayland] )
```

Comment:
Why the dependency? It doesn't seem like the package would build or work incorrectly if people don't have wayland installed.